src/devices/Mcp3008/samples/Mcp2008.sample.cs

```diff
@@ -19,37 +19,41 @@ static void Main(string[] args)
             // The SPI option is enabled in the sample by default, but you can switch
             // to the GPIO bit-banging option by switching which one is commented out.
             // The sample uses local functions to make it easier to switch between
-            // the two implementations
+            // the two implementations.
 
             // SPI implementation
-            Mcp3008 GetMCP3008WithSPI()
+            Mcp3008 GetMcp3008WithSpi()
             {
-                var connection = new SpiConnectionSettings(0,0);
-                connection.ClockFrequency = 1000000;
-                connection.Mode = SpiMode.Mode0;
+                Console.WriteLine("Using SPI protocol.");
+
+                var connection = new SpiConnectionSettings(0, 0)
+                {
+                    ClockFrequency = 1000000,
+                    Mode = SpiMode.Mode0
+                };
+
                 var spi = new UnixSpiDevice(connection);
                 var mcp3008 = new Mcp3008(spi);
                 return mcp3008;
             }
 
-
-            // the GPIO (via bit banging) implementation
-            Mcp3008 GetMCP3008WithGPIO()
+            // GPIO (via bit banging) implementation
+            Mcp3008 GetMcp3008WithGpio()
             {
+                Console.WriteLine("Using GPIO pins.");
                 var mcp3008 = new Mcp3008(18, 23, 24, 25);
                 return mcp3008;
             }
 
-            Mcp3008 mcp = GetMCP3008WithSPI();
-            // Uncomment next line to use Gpio instead.
-            //Mcp3008 mcp = GetMCP3008WithGPIO();
+            var mcp = GetMcp3008WithSpi();
```

Comment:
I actually prefer having the type instead of var. Usually we try to avoid var in order to make the code more readable, although in this particular case it is pretty obvious because of the method name.

Reply:
@joperezr you presumably follow the style guidelines of the corefx repo (unless you want to write your own) which allow car only if the type on the right side is obvious, basically it's a constructor

Reply:
I rarely use var in my day job, but wanted to try and go by the guidelines.  I was on the fence using var here, but the function included type in name.  So should I leave as var or Mcp3008?